sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -2986,7 +2986,7 @@ var setParentClass = function(child, parent) {
             var size = reference.rows * reference.cols;
             for (var i = 0; i < reference.rows; i++)
                 for (var j = 0; j < reference.cols; j++)
-                    ret.set(i, j, output[size * index + j * reference.cols + i]);
+                    ret.set(i, j, output[size * index + j * reference.rows + i]);
```

Comment:
Yeah, It depends on the layout. I mean, in the shader, it is in column-major order, but in JS (like c/c++/java), array is always in row-major order. 
Here comes the example. Let's take outerproduct(vec2, vec3) for an example too. The result would be
c0 * r0,     c0 * r1,      c0 * r2
c1 * r0,     c1 * r1,      c1 * r2 

In shader, the layout of the index would be (colum-majored, treat it located in 1 dimension array): 
0, 2, 4
1, 3, 5.   
But in JS, it is in row-major order. So when we get the result from shader, the index should be 0, 2, 4, 1, 3, 5. Exactly the same with the index after my patch applied. 

Please correct me if I am wrong. :)